bin/run_ascat.r

```diff
@@ -0,0 +1,58 @@
+#!/bin/env Rscript
+args = commandArgs(trailingOnly=TRUE)
+if(length(args)<6){
+    stop("No input files supplied\n\nUsage:\nRscript run_ascat.r tumor_baf tumor_logr normal_baf normal_logr tumor_sample_name baseDir gcfile\n\n")
+} else{
+    tumorbaf = args[1]
+    tumorlogr = args[2]
+    normalbaf = args[3]
+    normallogr = args[4]
+    tumorname = args[5]
+    baseDir = args[6]
+    gcfile = args[7]
+}
+
+source(paste(baseDir,"/scripts/ascat.R", sep=""))
+
+if(!require(RColorBrewer)){
+    source("http://bioconductor.org/biocLite.R")
+    biocLite("RColorBrewer", suppressUpdates=TRUE, lib="$baseDir/scripts")
+    library(RColorBrewer)
+}
+options(bitmapType='cairo')
+
+#Load the  data
+ascat.bc <- ascat.loadData(Tumor_LogR_file=tumorlogr, Tumor_BAF_file=tumorbaf, Germline_LogR_file=normallogr, Germline_BAF_file=normalbaf)
+
+#GC wave correction
+ascat.bc = ascat.GCcorrect(ascat.bc, gcfile)
```

Comment:
My feeling is that we do not have a GC file for GRCh37 (@malinlarsson knows it better).

Reply:
We do have one.
It's waiting to be sync with the other files on iGenomes, and I do think it was on UPPMAX for a long time

Reply:
Yes, we do have it.